astropy/utils/iers/iers.py

```diff
@@ -443,5 +473,49 @@ def pm_source(self, i):
         """Set PM source flag for entries in IERS table"""
         return np.ones_like(i) * FROM_IERS_B
 
+class IERS_Auto(IERS_A):
```

Comment:
Yeah, I thought that might be your thinking, and that makes sense.

If it were possible I would change the existing hierarchy such that there's some hidden `_IERSBase` class that is the same as the current `IERS` class, and what you're calling `IERS_Auto` would become just `IERS`.  I wonder if it isn't too late to make that change since direct use of `IERS` isn't particularly advertised in the docs.